packages/@o3r/create/src/index.ts

```diff
@@ -115,7 +115,7 @@ const isNgNewOptions = (arg: string) => {
 };
 
 const createNgProject = () => {
-  const { error } = spawnSync(process.execPath, [binPath, 'new', ...args.filter(isNgNewOptions)], {
+  const { error } = spawnSync(process.execPath, [binPath, 'new', ...args.filter(isNgNewOptions), '--skip-install'], {
```

Comment:
why `--skip-install`? Your issue was not regarding `--no-commit`?

Reply:
The problem I had was the fact that the update of gitignore done by 'ng add o3r/core' was after the `install` task done by '@angular/schematics'. Looking closely, the install task is not needed in the context of o3r/create. 
So we speed a bit the generation and I don;t have the problem with installation files not being git ignored.